doc/Language/faq.pod6

```diff
@@ -299,6 +299,77 @@ dependencies within a single compilation unit (e.g., file) are possible
 through stubbing. Therefore another possible solution is to move
 classes into the same compilation unit.
 
+
+=head1 Common operations
+
+=head2 String: How can I parse and get a L<number|/language/numerics> from a L<string|/language/type/Str>
+
+Use the L<+ prefix|/language/operators#prefix_+>:
+
+=begin code :ok-test<dd>
+say "42.123456789123456789";  # OUTPUT: «42.123456789123456789␤»
+say +"42.4e-2";               # OUTPUT: «0.424␤»
+=end code
+
+This method can numify any string you could enter as a L<litteral number|/language/syntax#Number_literals>.
+L<val|/routine/val> routine converts it to L<allomorph|/language/glossary#Allomorph>.
+L<unival|/routine/unival> routine converts one unicode codepoint.
+
+=head2 String: How can I check if a string contains a substring and if so, how can I get indices of matches
+
+Use L<.contains|/type/Str#method_contains> or L<.indices|/type/Str#method_indices>:
+
+=begin code :ok-test<dd>
+"az and az and az again".contains("az"); # OUTPUT: «True␤»
+"az and az and az again".indices("az");  # OUTPUT: «(0 7 14)␤»
+=end code
+
+=head2 String: How can I get the hexadecimal representation of a string
+
+First convert it to a L<Blob|/type/Blob> with L<.encode|/routine/encode>.
+
+=begin code :ok-test<dd>
+say "I ❤ 🦋".encode>>.base(16);  OUTPUT: «(49 20 E2 9D A4 20 F0 9F A6 8B)␤»
+=end code
+
+Note that L<.gist|/routine/gist> or L<.raku|/routine/perl> methods are your friends when L<debugging|/programs/01-debugging>:
+
```

Comment:
Er, what?

Reply:
"Are useful for variable introspection" better ?

Reply:
I don't see it related to the question. If there's anything implicit in the question about debugging, maybe you should say so explicitly.

Reply:
Just ask and I remove it. I have a reverse engineer background and this `.perl` is giving a fast view on the hex dump I wanted. Usually people looking for hexadecimal representation are analyzing binaries (Blobs).

Reply:
> Usually people

I admit, I take my personal case as a generality ...

Reply:
Not a friend of removing stuff; it's good work and it's done and why would you want to throw it away? My point is that maybe you should re-formulate the question a bit more pointedly... Or just craft another question for which that phrase is an answer. And also remember this is a FAQ, and F is *frequent*.

Reply:
Thanks for the review.

> I don't see it related to the question.

True, its VERY implicit but all your doc is very dense and I'm trying to follow this pattern. The secret relation is that `"I ❤ 🦋".encode.raku; # OUTPUT utf8.new(73,32,226,157,164,32,240,159,166,139)` so you can strop after `.encode`, if you are interested in bytes. The rest is for display.